editor/doc/doc_data.cpp

```diff
@@ -301,8 +301,14 @@ void DocData::generate(bool p_basic_types) {
 			if (E->get().name == "" || (E->get().name[0] == '_' && !(E->get().flags & METHOD_FLAG_VIRTUAL)))
 				continue; //hidden, don't count
 
-			if (skip_setter_getter_methods && setters_getters.has(E->get().name) && E->get().name.find("/") == -1)
-				continue;
+			if (skip_setter_getter_methods && setters_getters.has(E->get().name)) {
```

Comment:
For the reference, the check for `/` was properly trying to avoid skipping setters and getters for those properties: #17558
But `E->get()` here is the method, not the matching property, so this check was not doing anything here.